pipeline/filters/standard-output.md

```diff
@@ -1,37 +1,35 @@
-# Standard Output
```

Comment:
Can you add `# Standard Output`  again?
I think it will be same format like other plugins.